src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -60,9 +60,11 @@ def stdout_contents(wu):
     return f.read().rstrip()
 
 
-def dump_digest(output_dir, digest):
-  safe_file_dump('{}.digest'.format(output_dir),
-    '{}:{}'.format(digest.fingerprint, digest.serialized_bytes_length), mode='w')
+def write_digest(output_dir, digest):
+  safe_file_dump(
+    '{}.digest'.format(output_dir),
+    mode='wb',
```

Comment:
When you first started working on this PR, we did not specify what this should be. But since working on it, we merged a PR that changes this to `mode='w'` to fix unicode issues. This should go back to being `w`.

Reply:
Done!